test-it/ViaTests/ByteArrayString.via

```diff
@@ -0,0 +1,59 @@
+//define (true -43 'shut up about moon men')
```

Comment:
Delete this commented out line.

Reply:
Leave it, it’s innocuous and it made me laugh.

Reply:
Boo! Put it back!!